### ISSUE TITLE
[ADT] `aria_trajectory.csv` quaternions are wrong

### ISSUE THREAD

**Issue by brentyi** [facebookresearch/projectaria_tools#7]
It looks like the real component is stored in `qx_world_device`, while the imaginary ones are in `{qy,qz,qw}_world_device`.

I imagine this is because Sophus/Eigen store quaternions in `wxyz` order, but the MPS schema uses `xyzw`. Do you mind double-checking this?

Thanks!

**Comment by YLouWashU** [comment]
Oh yeah we have actually caught this issue internally. 

Some more context: so you are exactly right, the `aria_trajectory.csv` file has mis-labeled header due to different conventions of quaternions in different CPP and Python libs. However in our original release, we also loaded the data through `AriaDigitalTwinDataProvider` in a wrong order, which actually made the loaded quaternions to be correct. 

We have fixed this in [this commit](https://github.com/facebookresearch/projectaria_tools/commit/4930c775995d18175727a847857929a5fdcbf3e3), you can find out more details there. And we will be rolling out a new release of ADT dataset next week to fix the datasets, by which time you can re-download the ADT datasets where the trajectory files should have correct content. 

TLDR: for now, if you use our provided code to load this quaternion, the returned SE3 is still correct; but if you directly take this csv file and parse it yourself, then the header labeling is indeed wrong. But everything will be fixed by next week (hopefully Monday).

**Comment by brentyi** [comment]
Okay, that makes a lot of sense. I was trying to use the more general MPS APIs for exploring the dataset, but sounds like it makes sense to just switch to `AriaDigitalTwinDataProvider`. 🙂 

Looking forward to the updates!

**Comment by nickcharron** [comment]
Hi @brentyi, we have released [v1.1.0](https://github.com/facebookresearch/projectaria_tools/releases/tag/1.1.0) and updated the data which can now be re-downloaded using the same [instructions](https://facebookresearch.github.io/projectaria_tools/docs/open_datasets/aria_digital_twin_dataset/dataset_download) from the wiki. This should solve your issue. Closing this task, but please feel free to re-open if you find the problem persists.
